bigquery_etl/view/__init__.py

```diff
@@ -291,6 +280,30 @@ def has_changes(self, target_project=None):
                 print(f"view {target_view_id} will change: labels do not match")
                 return True
 
+        schema_file = Path(self.path).parent / "schema.yaml"
+        table_schema = Schema.from_bigquery_schema(table.schema)
+        # check schema based on schema file
+        if schema_file.is_file():
+            view_schema = Schema.from_schema_file(schema_file)
```

Comment:
nit: could be a property
```
@property/cached_property
def local_schema(self) -> Optional[Schema]:
    (some of this section)
    return None
```
Then the next section becomes `if self.local_schema is None`... etc.

Reply:
Good point.  I don't think this will run multiple times per object very often but probably doesn't hurt to add.